### ISSUE TITLE
Fix false positive reporting of `use-list` which is not meant to be list item

### ISSUE THREAD

**Issue by JounQin** [markuplint/markuplint#957]
- Markuplint version: `3.9.1`
- Parser lang: Svelte
- Node.js version: `18.16.0`
- OS: macOS

## Code Example

```svelte
<p>{count} * 2 = {doubled}</p>
```

## Steps To Reproduce

1.
2.

## The current behavior

warning

## The expected behavior

no warning

**Comment by YusukeHirao** [comment]
@JounQin Thanks for always reporting issues.

I'm hesitant to classify this as a bug at the moment. Markuplint currently doesn't determine whether a custom syntax block is variable expansion or the start of a special block. As per the current specification, we recommend disabling rules partially. 

However, there's room for improvement in our approach to custom syntax, so if you have any ideas, I'd love to hear them.
